app/assets/javascripts/oxalis/model/binary/data_cube.js

```diff
@@ -377,19 +377,22 @@ class DataCube {
         result += (1 << (8 * i)) * data[voxelIndex + i];
       }
 
-      if (mapping) {
-        const mappedValue = mapping[result];
-        if (mappedValue != null) {
-          return mappedValue;
-        }
-      }
-
-      return result;
+      return this.mapID(result, mapping);
     }
 
     return 0;
   }
 
+  mapID(id: number, mapping: ?MappingType): number {
```

Comment:
There already is a ` mapId` function in the data_cube :)

Reply:
:man_facepalming: